src/main/java/org/usfirst/frc/team1758/robot/subsystems/Vision.java

```diff
@@ -1,74 +1,52 @@
 package org.usfirst.frc.team1758.robot.subsystems;
 
 import org.usfirst.frc.team1758.robot.RobotMap;
+import org.usfirst.frc.team1758.robot.commands.ToggleLight;
+import org.usfirst.frc.team1758.robot.commands.ToggleLight.LightMode;
+
+
 import edu.wpi.first.wpilibj.CameraServer;
 import edu.wpi.first.wpilibj.Relay;
 import edu.wpi.first.wpilibj.Relay.Value;
 import edu.wpi.first.wpilibj.command.Subsystem;
-import edu.wpi.first.wpilibj.vision.VisionThread;
 import edu.wpi.cscore.AxisCamera;
-import edu.wpi.cscore.CvSink;
-import edu.wpi.cscore.CvSource;
-import edu.wpi.cscore.MjpegServer;
 import edu.wpi.cscore.UsbCamera;
 
 public class Vision extends Subsystem
 {
 	private boolean isLightOn;
-	private VisionThread frontVisionThread, backVisionThread;
+	private AxisCamera frontCamera;
 	private Relay cameraLightRelay;
-	AxisCamera frontCamera, backCamera;
 	UsbCamera frontCameraUSB, backCameraUSB;
 	CameraServer serverCamera;
-	MjpegServer mjpegServer;	
-	CvSink cvSink;
-	CvSource cvSource;
 	public Vision() 
 	{ 
     cameraLightRelay = new Relay(RobotMap.CAMERA_LIGHT_RELAY); 
-    isLightOn = false; 
+		serverCamera = CameraServer.getInstance();
+    isLightOn = false;
 		frontCamera = new AxisCamera("frontCamera", RobotMap.ipFrontCamera);
-		backCamera = new AxisCamera("backCamera", RobotMap.ipBackCamera);
 		configureFrontCamera();
-		serverCamera = CameraServer.getInstance();
-		frontCameraUSB = new UsbCamera("frontCameraUSB", 0);
-		//mjpegServer = new MjpegServer("cameraStream", 1181);
-		//mjpegServer.setSource(frontCameraUSB);
-		//Testing around with the CvSink
-		//cvSink = new CvSink("openCv_USBCamera_0");
-		//cvSink.setSource(frontCameraUSB);
-		//CameraServer.getInstance().addServer(mjpegServer);
-		serverCamera.addAxisCamera("frontCamera", "host");
-		//frontCameraUSB = CameraServer.getInstance().
-		//backCameraUSB = CameraServer.getInstance().addServer("backCameraUSB", port);
+		switchToFrontCamera();
 	} 
 	protected void initDefaultCommand() 
   { 
-    //setDefaultCommand(new ToggleLight(LightMode.ON)); 
-		//setDefaultCommand(new SwitchCamera(cameraInUse.TOGGLE));
-  } 
-	public void switchToBackCamera()
-	{
-		serverCamera.removeCamera("frontCamera");
-		serverCamera.addAxisCamera("backCamera", "host");
-	}
+		setDefaultCommand(new ToggleLight(LightMode.ON));
```

Comment:
Maybe a better way to do this? Either we wrote the command wrong or this is doing something I was expecting, but I only want this command to be executed once then stop not keep turning the light on. This may cause issues in the future.